From: Jeff Johnson <quic_jjohnson@quicinc.com>
Subject: lib/list_test: add the missing MODULE_DESCRIPTION() macro
Date: Fri, 31 May 2024 16:58:42 -0700

make allmodconfig && make W=1 C=1 reports:
WARNING: modpost: missing MODULE_DESCRIPTION() in lib/list-test.o

Add the missing invocation of the MODULE_DESCRIPTION() macro.

Link: https://lkml.kernel.org/r/20240531-md-lib-list-test-v1-1-50b79241cbb0@quicinc.com
Signed-off-by: Jeff Johnson <quic_jjohnson@quicinc.com>
Reviewed-by: David Gow <davidgow@google.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 lib/list-test.c |    1 +
 1 file changed, 1 insertion(+)

--- a/lib/list-test.c~list-test-add-the-missing-module_description-macro
+++ a/lib/list-test.c
@@ -1499,4 +1499,5 @@ static struct kunit_suite klist_test_mod
 
 kunit_test_suites(&list_test_module, &hlist_test_module, &klist_test_module);
 
+MODULE_DESCRIPTION("KUnit test for the Kernel Linked-list structures");
 MODULE_LICENSE("GPL v2");
_
